libs/base/pxtbase.h

```diff
@@ -1047,14 +1050,15 @@ template <size_t N> constexpr size_t _boxedStringLen(char const (&)[N]) {
 
 // strings defined here as used as (String)name
 #define PXT_DEF_STRING(name, val)                                                                  \
-    const BoxedStringLayout<_boxedStringLen(val)> name[1] = {                                  \
+    const BoxedStringLayout<_boxedStringLen(val)> name[1] = {                                      \
         {&pxt::string_inline_ascii_vt, _boxedStringLen(val) - 1, val}};
 
 // bigger value - less memory, but slower
 // 16/20 keeps s.length and s.charCodeAt(i) at about 200 cycles (for actual unicode strings),
 // which is similar to amortized allocation time
 #define PXT_STRING_SKIP_INCR 16 // needs to be power of 2; needs to be kept in sync with compiler
-#define PXT_STRING_MIN_SKIP 20  // min. size of string to use skip list; static code has its own limit
+#define PXT_STRING_MIN_SKIP                                                                        \
+    20 // min. size of string to use skip list; static code has its own limit
```

Comment:
new C code formatter?

Reply:
no; I suspect some manual changes without reformatting in the past...